### PR TITLE
Provide a config knob to disable collecting root Cgroup stats #2259

### PR DESCRIPTION
**Pull request by sashankreddya**
#2259

### CODE REVIEW

**Comment by sashankreddya** [comment]
@dashpole : Looks like some test eco system issue in pulling changes (pull-cadvisor-e2e).

**Comment by dashpole** [comment]
```
The following files are not properly formatted:
container/raw/factory.go container/raw/factory.go
```

**Comment by sashankreddya** [comment]
> ```
> The following files are not properly formatted:
> container/raw/factory.go container/raw/factory.go
> ```

Fixed it.

File: container/raw/handler.go [hunk 1]
```diff
@@ -227,6 +227,9 @@ func (self *rawContainerHandler) getFsStats(stats *info.ContainerStats) error {
 }
 
 func (self *rawContainerHandler) GetStats() (*info.ContainerStats, error) {
+	if *disableRootCgroupStats && isRootCgroup(self.name) {
```
Comment (dashpole) [inline container/raw/handler.go]: can we do this in CanHandleAndAccept in factory.go similar to the `docker_only` flag?

Comment (sashankreddya) [reply container/raw/handler.go]: Sure will test it and update you.

Comment (sashankreddya) [reply container/raw/handler.go]: @dashpole :

We cant do it in "CanHandleandAccept" currently as [this step](https://github.com/google/cadvisor/blob/master/manager/manager.go#L273) throws error that it failed to find the container ("/") as the [container creation step for "/"](https://github.com/google/cadvisor/blob/master/manager/manager.go#L268) skips creating the container with message 

> 

I0810 00:05:34.490103  103510 factory.go:170] Factory "raw" can handle container "/", but ignoring.
I0810 00:05:34.490112  103510 manager.go:908] ignoring container "/"

>

Comment (dashpole) [reply container/raw/handler.go]: Ok.  Thanks for looking into it.

**Comment by trallnag** [comment]
@dashpole documentation on `disable_root_cgroup_stats` option is still missing in https://github.com/google/cadvisor/blob/master/docs/runtime_options.md

**Pull request merged**